src/qt/and_breakpoint_window.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef AND_BREAKPOINT_WINDOW_HPP
+#define AND_BREAKPOINT_WINDOW_HPP
+
+#include <QDialog>
+#include <QtCore/QVariant>
+#include <QtWidgets/QApplication>
+#include <QtWidgets/QComboBox>
+#include <QtWidgets/QDialog>
+#include <QtWidgets/QDialogButtonBox>
+#include <QtWidgets/QGridLayout>
+#include <QtWidgets/QLabel>
+#include <QtWidgets/QSpacerItem>
+#include "../core/ee/emotion_breakpoint.hpp"
+
+
+
+class AndBreakpointWindow : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit AndBreakpointWindow(QWidget *parent = 0);
```

Comment:
To be explicit, `nullptr` should be used here, unless there is a specific reason 0 is being used?

```suggestion
    explicit AndBreakpointWindow(QWidget *parent = nullptr);
```